### ISSUE TITLE
Zen browser is slower than Firefox 130

### ISSUE THREAD

**Issue by lemon07r** [zen-browser/desktop#1411]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Not sure why but this browser is slower than the base browser. I tested Firefox 130 stable, against v1.0.0-a39 in speedometer 3.0. Exact same settings, environment, etc. All controls were kept to their minimums and consistent. Logged in, synced and set up both browsers the same way, then I tested both in private window, with nothing else open, so extensions should have been disabled. I thought maybe it was testing variance so I tried a couple times but Zen scored 16 pretty consistently, whereas regular firefox 130 scores 17.2. I'm using the optimized binaries for zen on windows. Firefox stable had no tweaks done to it.
![firefox130](https://github.com/user-attachments/assets/a7877e2e-b74b-4a7f-ba40-31bddd485d56)
![zen1.0.0a39](https://github.com/user-attachments/assets/480962a0-e233-4721-82fa-135287276ba5)

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0a39

### What platform are you seeing the problem on?

Windows

**Comment by mauro-balades** [comment]
[browser benchmarks are stupid](https://www.dedoimedo.com/computers/browser-speed.html). If you are on windows specially, there are tons of factors affecting zen like Windows Defender on simply a change on internet speed. And the difference from 16 to 17 is almost not noticeable on a day to day basis

**Comment by lemon07r** [comment]
> [browser benchmarks are stupid](https://www.dedoimedo.com/computers/browser-speed.html). If you are on windows specially, there are tons of factors affecting zen like Windows Defender on simply a change on internet speed. And the difference from 16 to 17 is almost not noticeable on a day to day basis

The example you linked uses sunspider to make its point. Speedometer is pretty different, it's not an all encompassing test, but it does test the most popular UI frameworks, like react, etc. Benchmarks are not stupid, they're just misunderstood, and I get where you are coming with this, but the fact is Zen is significantly slower than base firefox in this test. If you think benchmarks are stupid, how are you guys judging performance than? Because I definitely dont trust subjective human feedback, I could tell you that the browser feels slower to me, but what would anyone care how I feel? I could just be biased or influenced by some sort of placebo effect. This is a fairly consistent benchmark, and 1 is a very big difference when its 16 vs 17 its the same difference as 160 vs 170 if big numbers help you visualize the difference better, so I dont really get why we are trying to downplay this with just "almost not noticable on a day to day basis". Chromium based browsers for example gets around 17-20 in this same test  on my computer and I definitely notice the difference day to day. 

>  All controls were kept to their minimums and consistent.

My windows defender was disabled. This was also a fresh clean install of windows on my PC, with drivers, etc all updated just a few days ago. Quite literally nothing running in the background but the essentials. I also reran the test multiple times to see if it was just variance. I got exact 16.0 or 16.1 every single time. More importantly, firefox was run under the exact same conditions. I don't know why youre being adamant on finding other factors blame here. I'm as skeptical as the next person so I try to be very careful with this kind of testing. Im not even sure why internet speed was mentioned here, internet speed does not affect benchmarks like this, which run locally on your PC. but If it makes you feel better I was running this on 1.5gbps internet over a cat7 cable using 2.5gbps ports?

If you would really prefer subjective opinions over benchmarking even after all is said and done, Zen does feel a bit slower to me than firefox, both fresh installed under the same conditions, and I'm not just saying that to work backwards and have my point. Its not to the point where its unusable but its too the point where it is noticable and would bother me in a long term. I actually only did this testing cause I noticed a difference in performance. I wanted to see if it showed up in benchmark too. In regular use I do have antivirus enabled but I do run something lighter than windows defender, currently using mcafee since I had a year key for it laying around, and it does have a lower performance impact than defender. 

Personally I think it would be nice if performance was looked at and improved upon, since this is an alpha browser there's room for improvement. My main reason for moving from floorp back to firefox was that it started to feel sluggish next to regular old firefox. I know everyone is not the same, and maybe performance isnt a priority for this browser. In that case my apologies for opening up this issue, but I am a little surprised by the very "there's nothing wrong with the browser and the user is wrong" stance being taken here.

**Comment by mauro-balades** [comment]
We are constantly trying to improve on performance, but as the amount of features increases, it's normal that zen will take a bit more of usage

**Comment by Sadi58** [comment]
I heard about the Zen browser just yesterday, and immediately installed it (portable ZIP) in the same way as I use Firefox.
I very much liked what has been done to improve the UI of Firefox, and the especially the new feature to split the window vertically or horizontally to display two websites in the same browser window. 
I then went on to compare the performance of the two as well as Brave browser, which has been my default browser lately at browserbench.org.
The results were surprising - firstly, because it showed me that the performance of Firefox has gone up so much since I compared a number of web browsers some time ago.
However, I was sad to see that the results of Zen browser, which I would very much like to make my default web browser, were so much below both Firefox and Brave.
Same machine, same operating system (Debian Trixie with KDE Plasma 5), same configuration...
Nevertheless, I'll keep Zen installed and updated, using it whenever I need a split browser window, and ready to take the position of default web browser as soon as the current performance gap is overcome.
![Screenshot_2024-11-16_12-42-26](https://github.com/user-attachments/assets/df213a73-6bdb-4c1a-aef6-4d984516bbc8)
